test/unittests/execute_numeric_test.cpp

```diff
@@ -16,9 +17,12 @@ namespace
 {
 ExecutionResult execute_unary_operation(Instr instr, uint64_t arg)
 {
+    const auto& instr_type = get_instruction_type_table()[static_cast<size_t>(instr)];
```

Comment:
Why `size_t` and not `uint8_t`, given `Instr` is `uint8_t`?

Reply:
I guessed it is for the indexing type, but `uint8_t` would be more clear I guess. It fits `size_t` anyhow.